### PR TITLE
Ft api

### PR DESCRIPTION
**Pull request by petermwash**

### CODE REVIEW

**Comment by iankigen** [comment]
@petermwash Fix merge conflict and tests after effecting the feedback.

**Comment by petermwash** [comment]
@iankigen merge conflicts fixed.

**Comment by petermwash** [comment]
I just added more API endpoints

File: app/models/orders.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+class Order(object):
+
+	order_items = {}
+	
+	def __init__(self, order_owner_name, meal_name, quantity):
+		self.order_id = 1 #random.randint(1, 500)
+		self.order_owner_id = order_owner_name
+		self.meal_name = meal_name
+		self.quantity = quantity
+		#elf.order_items[meal_name] = quantity
```
Comment (domiebett) [inline app/models/orders.py]: You don't need commented out code.

File: app/models/users.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
+from flask import Flask
+from marshmallow import Schema, fields
+from passlib.apps import custom_app_context as pwd_context
+import random
+from itsdangerous import (TimedJSONWebSignatureSerializer
+                          as Serializer, BadSignature, SignatureExpired)
+
+
+app = Flask(__name__)
```
Comment (domiebett) [inline app/models/users.py:9]: Any reason why you have initialized the app twice, here and in the views.py?

**Review by domiebett — CHANGES_REQUESTED** [review]

File: app/models/orders.py (hunk 1 above)
Comment (petermwash) [reply app/models/orders.py]: Fixed

File: app/models/users.py (hunk 2 above)
Comment (petermwash) [reply app/models/users.py:9]: It was giving an import error when I tried to import the app from views.py but when I initialized it here it worked fine.

**Pull request merged**